.github/workflows/docker-build.yml

```
@@ -127,3 +127,100 @@ jobs:
+  #   v1.2.3-cpu-arm64   — immutable, on v*.*.* release tags
+  #   1.2-cpu-arm64      — rolling major.minor, on v*.*.* release tags
+  #   latest-cpu-arm64   — moving, on default-branch (main) builds
+  #   edge-cpu-arm64     — moving HEAD-of-main pointer
+  # The amd64 cpu/cuda/rocm variants stay on the github-hosted build-docker job
+  # above; only the CPU variant has an arm64 counterpart (CUDA/ROCm are
+  # x86_64-only today).
+  build-docker-arm64:
+    if: >-
+      github.event_name == 'push' ||
+      github.event.workflow_run.conclusion == 'success'
+    runs-on: [self-hosted, linux, arm64, graviton, hyprstream-merge-gate]
```

actionlint (1.7.12)

[error] label "graviton" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)